inscriptions/ISic000639.xml

```diff
@@ -34,6 +34,10 @@
      	       <name xml:id="SS" ref="https://orcid.org/0000-0003-3914-9569">Simona Stoyanova</name>
      	       <resp>standardisation of template and tidying up encoding</resp>
  	   </respStmt>
+                <respStmt>
+                    <name xml:id="FP">Francesca Prado</name>
+                    <resp>editing of EpiDoc</resp>
+                </respStmt>
```

Comment:
I see that this file has an EDR number also, which you could add: 141109
This isn't essential, but it is useful to do.